src/nfa/operations.cc

```diff
@@ -56,6 +56,126 @@ namespace {
         return lts_for_simulation.compute_simulation();
     }
 
+    bool vec_contains(std::vector<std::pair<State, State>> vec, std::pair<State, State> par){
```

Comment:
Furthermore, one should not create a custom function to search a container, unless they want to use some special non-standard search algorithm, but instead use `std::find`, or `<container_instance>.find() != <container_instance.end()>` or `<container_instance>.contains()` etc.

Reply:
This function no longer exists